schemas/2015-08-31-preview/Microsoft.ManagedIdentity.json

```diff
@@ -7,16 +7,6 @@
     "userAssignedIdentities": {
       "type": "object",
       "properties": {
-        "name": {
```

Comment:
Are these updates result of running https://github.com/Azure/azure-resource-manager-schemas/actions/workflows/generate-types.yml on your branch?

Reply:
nvm, it is.